tensorflow_datasets/scripts/templates/dataset.mako.md

```diff
@@ -280,3 +280,10 @@ ${display_builder(builder, all_sections)}
 % else:
 ${display_all_builders(config_builders)}
 % endif
+
+%if builder.info.name:
```

Comment:
Filtering on the name is temporary, how can we know if the dataset has an example without changing the builder? Would checking if the file exist be enough? @Conchylicultor

Reply:
Yes, checking for file existence is good enough.